libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1099,7 +1099,11 @@ private void replaceDrawable(Drawable newDrawable) {
         }
     }
 
-    @Override public void appendMediaFiles(Map<String, MediaFile> mediaList) { }
+    @Override public void appendMediaFiles(Map<String, MediaFile> mediaList) {
```

Comment:
@malinajirka Can you let me know why this change was necessary? I couldn't find if this was moved from somewhere or fixing a bug. I just didn't want to skip reviewing this, I am sure it's necessary :)

Reply:
Before this PR, we'd call either `appendMediaFiles` or `appendMediaFile` depending on the `allowMultipleSelection`. I removed that flag and decided to always call `appendMediaFiles` even when I have only one media file. However, this method wasn't implemented in AztecEditorFragment -> therefore I added this simple implementation.